scripts/release-bit-verification.sh

```diff
@@ -67,10 +70,9 @@ fi
 git clone $REPO_URL odo && cd $WORKING_DIR/odo && git checkout "v$VERSION"
 
 #Run tests
-make test-integration-devfile
-make test-integration
-make test-e2e-all
-make test-cmd-project
+make test-e2e
 
 # Cleanup
 rm -rf /tmp/odo /tmp/usr
```

Comment:
I think we can just remove `"$WORKING_DIR"`:
```suggestion
rm -rf "$WORKING_DIR"
```